word_count.py

```diff
@@ -0,0 +1,15 @@
+def word_count(word_list, columns, rows):
+	word_count = 0
+	word_iterator = 0
```

Comment:
`word_i` and `row_i` would probably suffice, but that's probably just nitpicking.